google_drive/tests/unit/test_docs.py

```diff
@@ -0,0 +1,240 @@
+""" Tests for google drive components """
+import json
+import unittest
+
+from webob import Request
+from mock import Mock
+
+from workbench.runtime import WorkbenchRuntime
+from xblock.runtime import KvsFieldData, DictKeyValueStore
+from xblock.fields import ScopeIds
+
+from google_drive import GoogleDocumentBlock, GoogleCalendarBlock
+
+from nose.tools import assert_equals, assert_in
+
+
+def generate_scope_ids(runtime, block_type):
+    """ helper to generate scope IDs for an XBlock """
+    def_id = runtime.id_generator.create_definition(block_type)
+    usage_id = runtime.id_generator.create_usage(def_id)
+    return ScopeIds('user', block_type, def_id, usage_id)
+
+
+def make_request(body, method='POST'):
+    """ helper to make a request """
+    request = Request.blank('/')
+    request.method = 'POST'
+    request.body = body.encode('utf-8')
+    request.method = method
+    return request
+
+
+class TestGoogleDocumentBlock(unittest.TestCase):
+    """ Tests for GoogleDocumentBlock """
+
+    @classmethod
+    def make_document_block(cls):
+        """ helper to construct a GoogleDocumentBlock """
+        runtime = WorkbenchRuntime()
+        key_store = DictKeyValueStore()
+        db_model = KvsFieldData(key_store)
+        ids = generate_scope_ids(runtime, 'google_document')
+        return GoogleDocumentBlock(runtime, db_model, scope_ids=ids)
+
+    def test_document_template_content(self):  # pylint: disable=no-self-use
+        """ Test content of GoogleDocumentBlock's rendered views """
+        block = TestGoogleDocumentBlock.make_document_block()
+        block.usage_id = Mock()
+
+        student_fragment = block.render('student_view', Mock())
+        # pylint: disable=no-value-for-parameter
+        assert_in('<div class="google-docs-xblock-wrapper"', student_fragment.content)
+        assert_in('Google Document', student_fragment.content)
+        assert_in(
+            (
+                'https://docs.google.com/presentation/d/1x2ZuzqHsMoh1epK8VsGAlanSo7r9z55ualwQlj-ofBQ'
```

Comment:
Lots of URLs and hashes throughout this test file -- do they correspond w/ the other URLs/hashes in the code?  Definitely seems like we should push these bits of data to either config or at least a helper method, property, constant...